packages/web-app-files/src/views/spaces/Projects.vue

```diff
@@ -200,17 +302,40 @@ export default {
   &-card {
     box-shadow: none !important;
 
+    .space-context-btn {
+      z-index: 999;
+    }
+
     .oc-card-media-top button {
       top: 0;
       right: 0;
     }
   }
 
+  &-card.state-trashed {
+    .space-image,
+    .space-default-image > svg {
+      filter: grayscale(100%);
```

Comment:
Did you resolve the issue from last week here?

Reply:
Yep

Reply:
Awesome!